packages/workflow/src/records/fhir.ts

```diff
@@ -187,9 +165,8 @@ export async function withPractitionerDetails<T extends Task>(
     {
       type: 'document',
       resourceType: 'Bundle',
-      entry: [practitioner, office, officeLocation].map((r) =>
-        resourceToSavedBundleEntry(r)
-      )
+      // @TODO: Do we need office location here?
+      entry: [practitioner, office].map((r) => resourceToSavedBundleEntry(r))
```

Comment:
?

Reply:
Atleast the application works functionally after not doing this, but I haven't run E2E or gone through every flow

Reply:
The `findLastOfficeLocationFromSavedBundle` function need it I suppose but are we using it anywhere yet?

Reply:
Very good point. I kinda removed it from here but at the same time use that function in notification for example. I will think of an angle to tackle this.. 🤔

Reply:
We now fetch the hierarchy here. A discussion spawned from this where either the microservices themselves could fetch the full record with only passing them an id, OR fetching the full record here to not need to do the optimistical update. Not a refactor for this PR though.